pkg/cluster/manager.go

```diff
@@ -252,7 +252,8 @@ func (m *Manager) RestartCluster(clusterName string, options operator.Options) e
 func (m *Manager) ListCluster() error {
 	names, err := m.specManager.List()
 	if err != nil {
-		return perrs.AddStack(err)
+		log.Warnf("List cluster error %v", err)
+		// return perrs.AddStack(err)
```

Comment:
Why not returning here?

Reply:
Sorry, this is a mistake, will be reverted back.